src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2658,6 +2658,19 @@ bool primitive_inst::is_valid_fusion() const {
         if (fd.is_type<eltwise>())
             can_broadcast = ov::PartialShape::broadcast_merge_into(merged_shape, outer_dep_pshape, fd.typed_desc<eltwise>()->broadcast_spec);
 
+        // Check if broadcast happens more than single axis.
+        // Current FUSED_OP_LOAD macro cannot support broadcast on dynamic dimension.
+        if (can_broadcast == true && (merged_shape.is_static() && outer_dep_pshape.is_static()) &&
+            outer_dep.first->_is_dynamic == true && merged_shape.rank().get_length() == outer_dep_pshape.rank().get_length()) {
```

Comment:
>> outer_dep.first->_is_dynamic

Do we need this condition? even though it is not dynamic, the fusion is not allowed isn't it?

Reply:
You are right. the is_valid_fusion() is called in dynamic shape case. I removed it. Thanks!